app/bundles/EmailBundle/Tests/Controller/EmailControllerFunctionalTest.php

```diff
@@ -33,6 +33,26 @@ public function setUp(): void
         parent::setUp();
     }
 
+    /**
+     * Index should return status code 200.
+     */
+    public function testIndexActionWhenNotFiltered(): void
+    {
+        $this->client->request('GET', '/s/emails');
+        $clientResponse = $this->client->getResponse();
+        $this->assertSame(200, $clientResponse->getStatusCode(), 'Return code must be 200.');
```

Comment:
Tests like these are definitely good to have, but it has little to do with your code change. Only that it's not throwing any exceptions when there are no emails in the list. Can we actually create some emails and test that it shows the data you added to the table? Something like
```suggestion
        $email = $this->createEmail('ABC', 'template');
        $email->setDateAdded(new \DateTime('2020-02-07 20:29:02'));
        $this->em->persist($email);
        $this->em->flush();
        $this->em->clear();

        $this->client->request('GET', '/s/emails');
        $clientResponse = $this->client->getResponse();
        $this->assertSame(200, $clientResponse->getStatusCode(), 'Return code must be 200.');
        $this->assertStringContainsString('2020-02-07 20:29:02', $clientResponse->getContent());
```
⬆️  is untested and will fail as there will be different date time format and we have to test also for the user and date modified but you get the idea.

Reply:
Done!